blog/_posts/2017-10-01-what-a-whirlwind.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Team Impact Devs - What a Whirlwind!
+layout: post
+created_at: Sun Oct 1 2017
+permalink: blog/_posts/2017-10-01-What-a-Whirlwind
+author: Team ImpactDevs
+twitter: teamImpactdevs
+---
+After a hectic three months, it's hard to believe that our Rails Girls Summer of Code experience is coming to an end.  This past week we celebrated with a few milestones.  The first was giving a presentation to our host company, [info.nl](https://www.info.nl/) sharing all of our learnings and a project demo.  We also threw an End of Summer Celebration for all of our coaches and friends in typical Dutch fashion - with bier and borel by the canal.
+
+Preparing for the presentation and bringing together our RGSoC community gave us a great chance to reflect on everything we have learned and accomplished over the past few months:
+
+<br>
+
+## What did we build?
+
+Our project requirements had two components:  
+
+1) A notification center where website admins can create and save notification messages to Worldbrain's 4,000 users.  Notifications can consist of anything from product updates, security alerts and new features.  Notifications are then saved to a mongo db, and made available as an API.  
+
+A react app that renders both a form and list of notifications previously created and sent.  
+
+2) An addition to the chrome extension that shows users there are new notifications via a badge on the extension icon, and list that shows all the notifications, all in React.  All unread notifications are bolded, and when it has been read it then becomes un-bolded.  Each user's read & unread notifications are saved and updated locally in the browser using PouchDB.
+
+![teamImpactdevs](/img/blog/2017/Worldbrain Project.png)
+<div class="image-credits"> What we built this summer</div>
```

Comment:
please add credits for the image (i.e. who created it or where it comes from)

Reply:
We took this photo and we made the stickers